builder/vmware/common/driver.go

```diff
@@ -642,8 +647,46 @@ func GetOVFTool() string {
 	return ovftool
 }
 
+func CheckOvfToolVersion(ovftoolPath string) bool {
+	output, err := exec.Command(ovftoolPath, "--version").CombinedOutput()
+	if err != nil {
+		return false
+	}
+	versionOutput := string(output)
+
+	versionRegex, err := regexp.Compile(`\d+\.\d+\.\d+`)
+	if err != nil {
+		log.Fatalf("failed to compile version regex: %v", err)
+	}
+
+	fields := strings.Fields(versionOutput)
+	for _, field := range fields {
+		if versionRegex.MatchString(field) {
+			currentVersion, err := version.NewVersion(field)
+			if err != nil {
+				log.Fatalf("failed to parse version '%s': %v", field, err)
```

Comment:
I would suggest not using `Fatalf` in functions generally, as the plugin crashes, which could cascade upwards.

Reply:
Resolved in 939a252.